integration/src/test/scala/org/knora/webapi/e2e/admin/GroupsADME2ESpec.scala

```diff
@@ -256,35 +256,36 @@ class GroupsADME2ESpec extends E2ESpec with GroupsADMJsonProtocol {
         )
       }
 
-      "DELETE a group" in {
-        val groupIriEnc = java.net.URLEncoder.encode(newGroupIri.get, "utf-8")
-        val request = Delete(baseApiUrl + "/admin/groups/" + groupIriEnc) ~> addCredentials(
-          BasicHttpCredentials(imagesUser01Email, testPass)
-        )
-        val response: HttpResponse = singleAwaitingRequest(request)
-        logger.debug(s"response: {}", response)
-        response.status should be(StatusCodes.OK)
-
-        val groupInfo: GroupADM = AkkaHttpUtils.httpResponseToJson(response).fields("group").convertTo[GroupADM]
-
-        groupInfo.name should be("UpdatedGroupName")
-        groupInfo.descriptions should be(Seq(StringLiteralV2("UpdatedGroupDescription", Some("en"))))
-        groupInfo.project should be(SharedTestDataADM.imagesProjectExternal)
-        groupInfo.status should be(false)
-        groupInfo.selfjoin should be(false)
-
-        clientTestDataCollector.addFile(
-          TestDataFileContent(
-            filePath = TestDataFilePath(
-              directoryPath = clientTestDataPath,
-              filename = "delete-group-response",
-              fileExtension = "json"
-            ),
-            text = responseToString(response)
-          )
-        )
-
-      }
+//      "DELETE a group" in {
```

Comment:
**todo**: Remove commented out code or make it work.

Reply:
[12de345](https://github.com/dasch-swiss/dsp-api/pull/3075/commits/12de3453bea0c4b09ff9217ad4cfff4772ca9f3c)